Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: lint
+lint: golangci-lint
+	$(GOLANGCI_LINT) run
+
+# install golangci-lint
+GOLANGCI_LINT = $(shell pwd)/bin/golangci-lint 	# to this path
+GOLANGCI_LINT_VERSION ?= v1.49.0 				# at this version
```

Comment:
nit: could we find a way to define the version in only one place and that be used elsewhere? (makefile, gha)

Reply:
What do you think about just running the Makefile target as the action? E.g. running `make lint` instead of running the `golangci/golangci-lint-action@v2`?

Reply:
I like that idea; I don't see the benefit of the GHA if we are not providing a way to lint.